### PR TITLE
Cleanup type declarations to be symmetric with other function definitions.

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
6bdb3f9 is in #206. Will rebase once merged.

File: zeroeq/http/types.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+/* Copyright (c) 2017, Human Brain Project
+ *                     Stefan.Eilemann@epfl.ch
+ */
+
+#pragma once
+
+#include <zeroeq/types.h>
```
Comment (rdumusc) [inline zeroeq/http/types.h]: not used?

**Comment by eile** [comment]
Yes, we had discussed:
* the function typedefs for subscriber did not follow the same pattern
* I feel that one file, one type should be a rule with the exceptions of typedefs in types.h

Imo the only alternative would be making this sub-types of the class, which is done in other places.

**Comment by eile** [comment]
Ping

File: zeroeq/http/types.h (hunk 1 above)
Comment (eile) [inline zeroeq/http/types.h]: Is used, at least for some basic types like uint128_t

**Pull request merged**